helm/ffc-template-node/values.yaml

```diff
@@ -0,0 +1,43 @@
+environment: development
+name: ffc-template-node
+namespace: ffc-demo
+workstream: ffc-demo
+image: ffc-template-node
+
+labels: {}
+
+containerConfigMap:
+  name: ffc-template-node-config-map
+
+deployment:
+  restartPolicy: Always
+  priorityClassName: low
+  replicas: 1
+  minReadySeconds: 5
+  redeployOnChange: update this field to force a change
+  runAsNonRoot: true
+  runAsUser: 1000
+
+container:
+  imagePullPolicy: IfNotPresent
+  requestMemory: 60Mi
+  requestCpu: 60m
+  limitMemory: 120Mi
```

Comment:
Might be best if we set these values to be guaranteed as that what we'd strive for in production over burstable.  ie. request and limit are the same.  Guaranteed pods have higher priority in k8s, it's also easier for k8s to more effectively allocate them to a running now.

However, it's going to be rare on starting a project that you know what the demand pattern will be and only through profiling will you learn.  So there is an argument that it should start as burstable and evolve to guaranteed (where appropriate).  But I think the other way round is better.

Reply:
Agreed, done